config/locales/en.yml

```diff
@@ -266,6 +266,7 @@ en:
       email: Email
       github: GitHub
       income_report: Income report
+      # Please use the language name of this locale as the translation.
```

Comment:
FYI this comment will be removed via `bundle exec i18n-tasks normalize`.

Reply:
Yes, do you have any advice about how to remind other people about the purpose of this `language` key?

Reply:
We could change the key name to make it more obvious. `language_in_native_tongue` or something like that.

Reply:
I am not sure about this, but I changed the key name to `language_name_of_locale`.
How do you think? 😀